### ISSUE TITLE
ERROR: Input masses are not defined for source 'u4.uddt'!

### ISSUE THREAD

**Issue by Julianfritzsche2** [santiago-sanitation-systems/Santiago.jl#30]
Error for: 
```Julia
input_masses = Dict("Dry.toilet" => Dict("phosphor" => 0.548,
                                         "nitrogen" => 4.550,
                                         "totalsolids" => 32.12,
                                         "water" => 547.1),
                    "Pour.flush" => Dict("phosphor" => 0.548,
                                         "nitrogen" => 4.55,
                                         "totalsolids" => 32.12,
                                         "water" => 1277.1),
                    "Cistern.flush" => Dict("phosphor" => 0.548,
                                            "nitrogen" => 4.55,
                                            "totalsolids" => 32.12,
                                            "water" => 22447.1),
                    # Urine diversion dry toilet
                    "Uddt" => Dict("phosphor" => 0.548,
                                   "nitrogen" => 4.55,
                                   "totalsolids" => 32.12,
                                   "water" => 547.1)
                    )
```
gives out an error when running `massflow_summary!` saying, that input masses are not defined for "u1.cistern.flush", "u2.dry.toilet" etc...

Renaming the sources for input masses solves the problem:
```Julia
input_masses = Dict("u3.dry.toilet" => Dict("phosphor" => 0.548,
                                         "nitrogen" => 4.550,
                                         "totalsolids" => 32.12,
                                         "water" => 547.1),
                    "u2.pour.flush" => Dict("phosphor" => 0.548,
                                         "nitrogen" => 4.55,
                                         "totalsolids" => 32.12,
                                         "water" => 1277.1),
                    "u1.cistern.flush" => Dict("phosphor" => 0.548,
                                            "nitrogen" => 4.55,
                                            "totalsolids" => 32.12,
                                            "water" => 22447.1),
                    # Urine diversion dry toilet
                    "u4.uddt" => Dict("phosphor" => 0.548,
                                   "nitrogen" => 4.55,
                                   "totalsolids" => 32.12,
                                   "water" => 547.1)
                    )
```

This error did not occur in the versions before, so either the code or the names in the tech library changed? Changing the code might be much easier, so minimal example has to be adjusted as well.

**Comment by scheidan** [comment]
Yes, the names of the sources in the dict of the input masses have to match the names in the techfile, this is not an error on the Santiago side

In the Readme I have used the names of the very small techfile that comes with Santiago. This is only used for testing (e.g. t check if your installation works).

It would be a good idea to define matching 'default" input masses in the https://github.com/santiago-sanitation-systems/Sanitation-technology-library repository.

**Comment by Julianfritzsche2** [comment]
Okay, but since the [tech library](the https://github.com/santiago-sanitation-systems/Sanitation-technology-library) contains the naming with e.g. "u1.cistern.flush" I changed the runfile with the input masses accordingly. 

I also created a .json file that could serve as a "default" input masses file. But I could not get it to read in properly again. But I think it would be a good idea to have the tech_library.json, case_template.json and default_input_masses.json on the github repository so all the files can be downloaded from there and edited via a .json editor.
